crds/v1alpha1/src/main/crds/analysisruns.argoproj.io

```diff
@@ -2783,7 +2783,7 @@ spec:
                               type: object
                           required:
                           - address
-                          - application
+                          - applicationConfig
```

Comment:
This definitely doesn't seem right. I don't think that CRDs changed since last time I pulled them.